packages/@blockprotocol/hook/src/types.ts

```diff
@@ -0,0 +1,36 @@
+import { MessageCallback } from "@blockprotocol/core";
+
+export type HookResponse = {
+  hookId: string;
+};
+
+export type HookData = {
+  node: HTMLElement | null;
+  type: string;
+  path: string;
+  hookId?: string | null;
```

Comment:
The JSON schema and the RFC said this should be `null` on first call, not omitted or null. We can change that but the JSON schema will need to be changed.
```suggestion
  hookId: string | null;
```

Reply:
I think it should be included always. Good catch.